ui/src/toolbar.test.tsx

```diff
@@ -13,22 +13,60 @@
 // limitations under the License.
 
 import React from 'react'
-import { render } from '@testing-library/react'
+import { fireEvent, render } from '@testing-library/react'
 import { View } from './toolbar'
 import * as T from './qd'
 
 const
   name = 'toolbar',
+  commandName = 'toolbar_command',
+  commandValue = 'toolbar_command_value',
+  commandLabel = commandName,
   toolbarProps: T.Card<any> = {
     name,
     state: { items: [] },
     changed: T.box(false)
+  },
+  toolbarPropsWithValueAttr: T.Card<any> = {
+    name,
+    state: { items: [{ name: commandName, value: commandValue, label: commandLabel }] },
+    changed: T.box(false)
+  },
+  toolbarPropsWithoutValueAttr: T.Card<any> = {
+    name,
+    state: { items: [{ name: commandName, label: commandLabel }] },
+    changed: T.box(false)
   }
```

Comment:
No need for these, use existing `toolbarProps` with base state: ` state: { items: [{ name: commandName, label: commandLabel }] },` and add value within specific test case as needed. Be careful to not change the original base props instance for further tests.